src/api/middleware/validators/validatorUtils.js

```diff
@@ -50,9 +52,22 @@ const isObjectId = (id) => {
     return true;
 };
 
+/**
+ * Checks if the concurrent offers of a given owner have not exceeded the defined limit
+ * @param {*} offerInstance
```

Comment:
change the name here as well and add documentation for the inner function (the second part of the arguments)